app/helpers/vita_min_form_builder.rb

```diff
@@ -63,16 +65,19 @@ def vita_min_state_file_select(
       method,
       label_contents(
         label_text,
-        options[:help_text],
+        has_help_text: !!options[:help_text],
         optional: options[:optional],
         ),
       class: label_class,
       )
     html_options_with_errors = html_options.merge(error_attributes(method: method))
 
+    help_html = help_text_html(options[:help_text], :method)
+
     html_output = <<~HTML
       <div class="form-group#{error_state(object, method)}">
         #{formatted_label}
+        #{help_html}
```

Comment:
can we have a screenshot in the PR description of what the select looks like with help text?

Reply:
Sure, though FWIW it looks like this select + help text is not in use anywhere in the app

Reply:
ohh didn't realize we don't have one. good to know!